examples/nextjs-spa/src/pages/recovery.tsx

```diff
@@ -84,8 +84,10 @@ const Recovery: NextPageWithLayout = () => {
         updateRecoveryFlowBody: values,
       })
       .then(({ data }) => {
+        // reset the form data completely
+        setFlow(null)
```

Comment:
Why is this needed, if we set it to the new flow immediately after? Wouldn't it be better to handle the loading state as such?

Reply:
resets the ui input field. for some reason setting the data directly will set the code field to the email value. 

recovery init -> submit email -> email is prefilled in code field

Reply:
Sounds like a bug, TBH. But, I guess it's a viable workaround.